src/mesh/Draco_Mesh_Builder.hh

```diff
@@ -0,0 +1,60 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   mesh/Draco_Mesh_Builder.hh
+ * \author Ryan Wollaeger <wollaeger@lanl.gov>
+ * \date   Friday, Jun 29, 2018, 09:55 am
+ * \brief  Draco_Mesh_Builder class header file.
+ * \note   Copyright (C) 2018 Los Alamos National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#ifndef rtt_mesh_Draco_Mesh_Builder_hh
+#define rtt_mesh_Draco_Mesh_Builder_hh
+
+#include <memory>
+
+namespace rtt_mesh_element {
+enum Geometry;
+}
+
+namespace rtt_mesh {
+
+// Forward declare mesh class
+class Draco_Mesh;
+
+//===========================================================================//
+/*!
+ * \class Draco_Mesh_Builder
+ *
+ * \brief Draco_Mesh unstructured mesh builder from reader.
+ *
+ * Reader is a template parameter ("FRT" = "Format Reater Type") to the builder
+ * class. Hence all readers that instantiate the template must have a common set
+ * of accessors.  In particular the reader must supply some functions in the
+ * RTT_Format_Reader class.
+ */
+//===========================================================================//
+
+template <typename FRT> class Draco_Mesh_Builder {
+private:
+  // >>> DATA
+
+  //! Pointer to reader
+  std::shared_ptr<FRT> reader;
+
+public:
+  //! Constructor
+  explicit Draco_Mesh_Builder(std::shared_ptr<FRT> reader_);
```

Comment:
Please add `DLL_PUBLIC_mesh` for the constructor to fix the error

```
 tstDraco_Mesh_Builder.obj : error LNK2019: unresolved external symbol "public: __thiscall rtt_mesh::Draco_Mesh_Builder<class rtt_RTT_Format_Reader::RTT_Format_Reader>::Draco_Mesh_Builder<class rtt_RTT_Format_Reader::RTT_Format_Reader>(class std::shared_ptr<class rtt_RTT_Format_Reader::RTT_Format_Reader>)" (??0?$Draco_Mesh_Builder@VRTT_Format_Reader@rtt_RTT_Format_Reader@@@rtt_mesh@@QAE@V?$shared_ptr@VRTT_Format_Reader@rtt_RTT_Format_Reader@@@std@@@Z) referenced in function "void __cdecl build_cartesian_mesh_2d(class rtt_c4::ParallelUnitTest &)" (?build_cartesian_mesh_2d@@YAXAAVParallelUnitTest@rtt_c4@@@Z) [C:\projects\build\src\mesh\test\Ut_mesh_tstDraco_Mesh_Builder_exe.vcxproj]
```

Reply:
Thank you - this has been added.